cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Broker.java

```diff
@@ -583,6 +589,46 @@ void setReplicaLoad(TopicPartition tp, AggregatedMetricValues aggregatedMetricVa
     return brokerMap;
   }
 
+  /**
+   * Get detailed disk utilization information of the broker.
+   *
+   * @return The detailed disk utilization information.
+   */
+  public Map<String, Double> diskUtils() {
+    if (_diskByLogdir == null || _diskByLogdir.isEmpty()) {
+      return null;
+    }
+    // If information of replica placement over disk is not populated, return nothing.
+    if (_diskByLogdir.values().stream().mapToInt(disk -> disk.replicas().size()).sum() == 0) {
+      return null;
+    }
+    Map<String, Double> diskUtilMap = new HashMap<>(_diskByLogdir.size());
+    for (Map.Entry<String, Disk> entry : _diskByLogdir.entrySet()) {
+      diskUtilMap.put(entry.getKey(), entry.getValue().load());
+    }
+    return diskUtilMap;
+  }
+
+  /**
+   * Get detailed disk capacity information of the broker.
+   *
+   * @return The detailed disk capacity information.
+   */
+  public Map<String, Double> diskCapacities() {
+    if (_diskByLogdir == null || _diskByLogdir.isEmpty()) {
+      return null;
```

Comment:
remove the `_diskByLogdir == null` since it is always initiated in `Broker`'s constructor. it can be empty when `BrokerCapacityConfigResolver` does not return per-logdir disk capacity information. Not necessary for JBOD case.